frontend/subrequests/outline/outline.go

```diff
@@ -0,0 +1,146 @@
+package outline
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"text/tabwriter"
+
+	"github.com/moby/buildkit/frontend/gateway/client"
+	"github.com/moby/buildkit/frontend/subrequests"
+	"github.com/moby/buildkit/solver/pb"
+)
+
+const RequestSubrequestsOutline = "frontend.outline"
```

Comment:
This feels weird to me, since the `describe` request is of the form `"frontend.subrequests.describe"`. I get that that `describe` is meta about the other subrequests, but it means that we have the weird feature that a client querying on buildx has: `outline`, `targets` and `subrequests.describe`. I think we should be able to hide from the user that we have subrequests behind `--print`, at least on the buildx side - maybe this is something to solve on the client instead, not sure.

Reply:
Yes, this is bit weird. But I didn't want users to need to type in `--print subrequests.outline`. On the same time I don't want to break `subrequests.describe` because we already released that some time ago and maybe somebody is using it. I also think that `--print describe` printing the list of possible commands is a bit misleading(user would more likely expect it to describe their build).